### PR TITLE
Add missing tests on events for Popup and Overlay

### PR DESCRIPTION
**Pull request by DokaRus**
https://trello.com/c/VcvIjujp

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets/popup.tests.js [hunk 1]
```diff
@@ -1453,6 +1478,23 @@ QUnit.module('templates', () => {
...
+    QUnit.test('titleRendered event should be fired if was set thought method', function(assert) {
+        assert.expect(1);
+
+        const $element = $('#popup').dxPopup({
+            visible: true,
+            showTitle: true
+        });
+        const instance = $element.dxPopup('instance');
+
+        instance.on('titleRendered', function(e) {
```
Comment (alexander-kotov-dx) [inline testing/tests/DevExpress.ui.widgets/popup.tests.js:1490]: sinon?

Comment (DokaRus) [reply testing/tests/DevExpress.ui.widgets/popup.tests.js:1490]: There is no need to use it. I just make an assertion in this handler.

**Review by alexander-kotov-dx — APPROVED** [review]

File: testing/tests/DevExpress.ui.widgets/popup.tests.js (hunk 1 above)
Comment (alexander-kotov-dx) [reply testing/tests/DevExpress.ui.widgets/popup.tests.js:1490]: Ok

**Review by San4es — APPROVED** [review]
LGTM

**Pull request merged**